### ISSUE TITLE
Does not preserve multi-line inlined procedures (e.g. UserData, ZipFile)

### ISSUE THREAD

**Issue by yingw787** [awslabs/aws-cloudformation-template-formatter#14]
I'd really appreciate if cfn-format did not re-format multi-line UserData or ZipFile statements. A resource like `AWS::AutoScaling::LaunchConfiguration` with property `UserData` gets formatted like this:

```yaml
      UserData: !Base64
        "Fn::Sub": "#!/bin/bash\nyum install -y aws-cfn-bootstrap\n/opt/aws/bin/cfn-init -v --region ${AWS::Region} --stack ${AWS::StackName} --resource ContainerInstances\n/opt/aws/bin/cfn-signal -e $? --region ${AWS::Region} --stack ${AWS::StackName} --resource ECSAutoScalingGroup\n\nexec 2>>/var/log/ecs/ecs-agent-install.log\nset -x\nuntil curl -s http://localhost:51678/v1/metadata\ndo\n   sleep 1\ndone\ndocker plugin install rexray/ebs REXRAY_PREEMPT=true EBS_REGION=us-west-2 --grant-all-permissions\nstop ecs \nstart ecs\n"
```

It might be better for machine-based parsing to use explicit characters, but it makes `git`-based diffs more difficult, and it's less legible to read (which is why I'm personally using YAML instead of JSON in the first place, because it supports indentation-as-code). Since the machine already has access to the full file to be reformatted using `--write`, would it be possible to ignore `UserData` or `ZipFile` arguments, or to re-format it as above and then unpack for pretty formatting afterwards?

**Comment by rburchill** [comment]
A related issue I'm seeing is `"Fn::Sub": |-` being transformed to `"Fn::Sub": |`. Subsequent formats then complain about the user data. I'm also prioritizing readability.

**Comment by stilvoid** [comment]
Fixed in 9856b93fb822f0eab6dd4917f6be33b6eb8435e4. I'll be making a new release shortly.
